### ISSUE TITLE
Training dataset with both HR/LR?

### ISSUE THREAD

**Issue by gutho877** [thunil/TecoGAN#47]
Following on from [this](https://github.com/thunil/TecoGAN/issues/17) thread, I'm still unable to get this to work. I think I'm messing something up with the code in dataloader.py but I still can't figure out what for the life of me.

Could someone please dumb it down a little more for me and if possible show which part of which line of the code provided in that thread, has to be changed? I really need help with this.

Thanks.

**Comment by gutho877** [comment]
Nobody?

**Comment by buildist** [comment]
I needed to do this and I think got it working, here are the changes I made to dataloader.py: https://www.diffchecker.com/PyaP0wJ8

Change lines 185-197 depending on the structure of your dataset.

**Comment by alessiapacca** [comment]
@buildist did you changed parameters like


```
input_video_dir', '', 'The directory of the video input data, for training'
'input_video_pre', 'scene', 'The pre of the directory of the video input data'
'str_dir', 1000, 'The starting index of the video directory'
'end_dir', 2000, 'The ending index of the video directory'
'end_dir_val', 2050, 'The ending index for validation of the video directory'
'max_frm', 119, 'The ending index of the video directory'
```
? 
cause I don't understand what I have to change actually.
Do you haave a list of changes you  made on the parameters? Like `TrainingDataPath` and so on

**Comment by alessiapacca** [comment]
@buildist can I ask you some questions about the changes you made? I wrote you an email

**Comment by adbobes** [comment]
@buildist can you help me with the returned shape of the tensors? I'm trying to print the shapes to see the output of the LoadHR function, but I couldn't

**Comment by FrankLinxzx** [comment]
> I needed to do this and I think got it working, here are the changes I made to dataloader.py: https://www.diffchecker.com/PyaP0wJ8
> 
> Change lines 185-197 depending on the structure of your dataset.

thx  a lot
